tests/client/amo/sagas/testCategories.js

```diff
@@ -0,0 +1,70 @@
+import { hideLoading, showLoading } from 'react-redux-loading-bar';
+import { call, put, select, takeEvery } from 'redux-saga/effects';
+
+import categoriesSaga, { fetchCategories } from 'amo/sagas/categories';
+import { getApi } from 'amo/sagas/utils';
+import * as actions from 'core/actions/categories';
+import { categories as categoriesApi } from 'core/api';
+import { CATEGORIES_FETCH } from 'core/constants';
+
+
+const categories = {};
+
+describe('categoriesSaga', () => {
+  it('should get Api from state then make API request to categories', () => {
+    const fetchCategoriesGenerator = fetchCategories();
+
+    let next = fetchCategoriesGenerator.next();
```

Comment:
Totally. I saw that library too and indeed even made a mistake during a refactor forgetting to re-assign `next` to the next result of the generator.

I'll give it a go with that lib.